Production/govuk_ios/ViewControllers/TabBarController.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import Foundation
+
+class TabBarController: UITabBarController {
+    private var notificationCenter: NotificationCenter = .default
+    private var standardAppearance: UITabBarAppearance {
+        let appearance = UITabBarAppearance()
+        appearance.configureWithOpaqueBackground()
+        appearance.stackedLayoutAppearance.selected.titleTextAttributes = titleTextAttributes
+        appearance.stackedLayoutAppearance.normal.titleTextAttributes = titleTextAttributes
+        return appearance
+    }
+    private var titleTextAttributes: [NSAttributedString.Key: Any] {
+        return [
+            .font: UIFont.govuk.tab.font
+        ]
+    }
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        registerNotifications()
+        updateAppearance()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func registerNotifications() {
+        notificationCenter.addObserver(
+            self,
+            selector: #selector(updateAppearance),
+            name: UIAccessibility.boldTextStatusDidChangeNotification,
+            object: nil
+        )
```

Comment:
I believe you should remove observers whenever you add them to a class in a `deinit` block.

Reply:
You don't _need_ to, but I am happy to do it if we deem it better practice to do so.
https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver#discussion

Reply:
If we go that route, might be better to use a block, instead of selector